code/modules/mob/language.dm

```diff
@@ -276,6 +276,19 @@
 	syllables = list("squeak")
 	flags = RESTRICTED
 
+/datum/language/martian
+	name = LANGUAGE_MARTIAN
+	desc = "Complex warbles and burbles used by the odd squid people."
+	speech_verb = "burbles"
+	ask_verb = "blorbles"
+	exclaim_verb = "blurbs"
+	key = "@"
+	colour = "grey"
```

Comment:
Were you planning on adding a unique color for the martian language, or are you fine with leaving it the same color as the grey language?

Reply:
Didn't see the need to make a unique colour seeing how it's not ACK ACK ACK